### PR TITLE
feat(dashbords): Implement widget new design part 5

### PR DESCRIPTION
**Pull request by priscilawebdev**

### CODE REVIEW

File: static/app/views/dashboardsV2/data.tsx [hunk 1]
```diff
@@ -2,8 +2,10 @@ import {t} from 'app/locale';
 
 import {DashboardDetails} from './types';
 
+export const emptyDashboardId = 'default-overview';
+
 export const EMPTY_DASHBOARD: DashboardDetails = {
-  id: '',
+  id: emptyDashboardId,
```
Comment (dashed) [inline static/app/views/dashboardsV2/data.tsx:8]: I think this should be an empty string. `default-overview` is a reserved dashboard ID.

File: static/app/views/dashboardsV2/detail.tsx [hunk 2]
```diff
@@ -58,6 +60,37 @@ class DashboardDetail extends React.Component<Props, State> {
     window.removeEventListener('beforeunload', this.onUnload);
   }
 
+  checkStateRoute() {
```
Comment (dashed) [inline static/app/views/dashboardsV2/detail.tsx:63]: This is not used anywhere.

File: static/app/views/dashboardsV2/data.tsx [hunk 3]
```diff
@@ -2,8 +2,10 @@ import {t} from 'app/locale';
 
 import {DashboardDetails} from './types';
 
+export const emptyDashboardId = 'default-overview';
```
Comment (dashed) [inline static/app/views/dashboardsV2/data.tsx:5]: This dashboard id (`default-overview`) refers to one of the hardcoded pre-built dashboards that comes with widgets. So it's not actually empty.

File: static/app/views/dashboardsV2/detail.tsx (hunk 2 above)
Comment (priscilawebdev) [reply static/app/views/dashboardsV2/detail.tsx:63]: good catch

File: static/app/views/dashboardsV2/data.tsx (hunk 3 above)
Comment (priscilawebdev) [reply static/app/views/dashboardsV2/data.tsx:5]: I see...but I noticed that it's not possible to delete the pre-built dashboard too, so this is always the "default"

File: static/app/views/dashboardsV2/data.tsx (hunk 1 above)
Comment (priscilawebdev) [reply static/app/views/dashboardsV2/data.tsx:8]: maybe the name here is not quire correct, but according to below logic I think this should be the default dashboard ID

https://github.com/getsentry/sentry/blob/9daf9246c86dbd43c8f3d1e456a81ef0dea31de8/static/app/views/dashboardsV2/orgDashboards.tsx#L92

Comment (priscilawebdev) [reply static/app/views/dashboardsV2/data.tsx:8]: When creating a dashboard I pass an empty id https://github.com/getsentry/sentry/blob/d63dec406c78e4245999b41a456f97084507a4fb/static/app/views/dashboardsV2/detail.tsx#L221

File: static/app/views/dashboardsV2/data.tsx (hunk 3 above)
Comment (dashed) [reply static/app/views/dashboardsV2/data.tsx]: @priscilawebdev We allow users to delete the pre-built dashboard by using "tombstones"; by marking it as deleted for an org.

**Review by dashed — APPROVED** [review]

**Pull request merged**